packages/esm-patient-chart-app/src/patient-chart/chart-review/dashboard-view.scss

```diff
@@ -16,6 +16,10 @@
   margin: 1.3125rem;
 }
 
+:global(.cds--link) {
```

Comment:
This override is meant to be in [esm-styleguide overrides](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-styleguide/src/_overrides.scss)